source/elm-live.js

```diff
@@ -29,10 +30,22 @@ module.exports = (argv, options) => {
 
   // Display help
   if (args.help) {
-    spawnSync('man', [
+    const manProcess = spawnSync('man', [
       path.resolve(__dirname, '../manpages/elm-live.1'),
     ], { stdio: 'inherit' });
```

Comment:
Yeah, good point. Sounds like more than a quick 5-minute-fix 😕. I’m afraid I won’t have time to fix it until Saturday. If you’re in a hurry, feel free to take over though 🙂.